android/src/test/java/com/oakam/launchdarkly_flutter/LaunchdarklyFlutterPluginTest.java

```diff
@@ -0,0 +1,183 @@
+package com.oakam.launchdarkly_flutter;
+
+import com.launchdarkly.sdk.LDUser;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.flutter.plugin.common.MethodCall;
+
+public class LaunchdarklyFlutterPluginTest {
+
```

Comment:
Nice job writing tests for the Android side, thanks very much for that! :) Wish you had done the same for the Swift side :)

Reply:
I'm struggling with Swift tests 😬. Apparently they're supposed to be added to the example not the plugin itself. And I can't even import `launchdarkly_flutter` module for some reason